### ISSUE TITLE
Perl-5.34.1-GCCcore-11.3.0.eb fails due to missing of extension Variable-Magic-0.62.tar.gz

### ISSUE THREAD

**Issue by RainManreports** [easybuilders/easybuild-easyconfigs#16673]
Actually Variable-Magic-0.62.tar.gz isn't available anymore under https://www.cpan.org/authors/id/V/VP/VPIT/. The new version is 0.63. Since Perl-5.34.1-GCCcore-11.3.0.eb is a required dependency for Intel toolchain 2022a intel-2022a.eb fails, too.

Solution: change to version 0.63 in Perl-5.34.1-GCCcore-11.3.0.eb with new sha256 

ba4083b2c31ff2694f2371333d554c826aaf24b4d98d03e48b5b4a43a2a0e679

fixes the problem.

Best regards,
RainMan

**Comment by APN-Pucky** [comment]
When I try to run `eb --prefix /scratch --installpath /easybuild GCCcore-11.3.0.eb gompi-2022a.eb Python-3.10.4-GCCcore-11.3.0.eb  --robot --trace   && rm -rf /scratch/*`, it also fails (eb 4.6.2):
```  
  >> download failed: https://www.cpan.org/authors/id/V/VP/VPIT/Variable-Magic-0.62.tar.gz
  >> download failed: https://www.cpan.org/src/5.0/Variable-Magic-0.62.tar.gz
  >> download failed: https://sources.easybuild.io/p/Perl/Variable-Magic-0.62.tar.gz
== ... (took 33 secs)
== FAILED: Installation ended unsuccessfully (build directory: /scratch/build/Perl/5.34.1/GCCcore-11.3.0): build failed (first 300 chars): Couldn't find file Variable-Magic-0.62.tar.gz anywhere, and downloading it didn't work either... Paths attempted (in order): /easybuild/software/EasyBuild/4.6.2/easybuild/easyconfigs/p/Perl/p/Perl/extensions/Variable-Magic-0.62.tar.gz, /easybuild/software/EasyBuild/4.6.2/easybuild/easyconfigs/p/Perl (took 35 secs)
== Results of the build can be found in the log file(s) /tmp/eb-eeab3fso/easybuild-Perl-5.34.1-20221231.132430.LDWLT.log

```
As said above, this is due to:
https://github.com/easybuilders/easybuild-easyconfigs/blob/94bd19b2c0216508aacc2ab982833458e1763535/easybuild/easyconfigs/p/Perl/Perl-5.34.1-GCCcore-11.3.0.eb#L1379
This source has already been changed to `cpan.metacpan.org` in the develop branch like the other packages in the `Perl.eb`.

**Comment by boegel** [comment]
Indeed, this was fixed in #16611

**Comment by boegel** [comment]
To install using the fixed easyconfig, you can use `eb --from-pr 16611 Perl-5.34.1-GCCcore-11.3.0.eb`
